Add windows version installer (EN + PT) (#90)
ZupIT/docs-ritchie: content/en/Getting Started/Windows.md

---

To install **the latest version of Ritchie** on Windows, you have to download Ritchie's installer and run it on your terminal.

## Step 1: Download the installer

> You can download the Ritchie latest version using this link: [latest version](https://commons-repo.ritchiecli.io/latest/ritchiecli.msi)
> 
> Or download any version by pasting this URL on your navigator substituting the `{VERSION}` field according to [the project repository tags](https://github.com/ZupIT/ritchie-cli/tags):
> 

Reviewer: ```suggestion
You can download Ritchie:
- Using this link to the [latest version](https://commons-repo.ritchiecli.io/latest/ritchiecli.msi);
- Or any version you want: just paste the URL on your browser replacing the `{VERSION}` field according to [the repository's project tags](https://github.com/ZupIT/ritchie-cli/tags):

```

```url
https://commons-repo.ritchiecli.io/{VERSION}/installer/ritchiecli.msi
```

> Once you made it, follow the instructions on your terminal after running the `rit` command.

Reviewer: ```suggestion
When you finish, follow the instructions on your terminal after running the `rit` command.
```
